src/services/fileStorage/proxy-service.js

```diff
@@ -33,16 +33,12 @@ const {
 	FILE_PREVIEW_SERVICE_URI,
 	FILE_PREVIEW_CALLBACK_URI,
 	ENABLE_THUMBNAIL_GENERATION,
-	FILE_SECURITY_CHECK_SERVICE_URI,
 	FILE_SECURITY_CHECK_MAX_FILE_SIZE,
-	FILE_SECURITY_SERVICE_USERNAME,
-	FILE_SECURITY_SERVICE_PASSWORD,
-	ENABLE_FILE_SECURITY_CHECK,
-	API_HOST,
 	SECURITY_CHECK_SERVICE_PATH,
 } = require('../../../config/globals');
+const { Configuration } = require('@schul-cloud/commons');
 
-const FILE_SECURITY_CHECK_CALLBACK_URI = url.resolve(API_HOST, SECURITY_CHECK_SERVICE_PATH);
+const FILE_SECURITY_CHECK_CALLBACK_URI = url.resolve(Configuration.get('API_HOST'), SECURITY_CHECK_SERVICE_PATH);
```

Comment:
set this value on request level, not global if possible

Reply:
you are right. without this globally, we won't need the docker-compose changes either